src/Livewire/Cart/Cart.php

```diff
@@ -209,8 +209,9 @@ public function updatedLoadWatchlist(): void
                 ->with('products')
```

Comment:
why eager load 'products' if they are not used? Better load cartItems:id,product_id,amount.
Then you can do ->cartItems->map().